brjs-sdk/sdk/libs/javascript/br-i18n/src/br/i18n/I18nStore.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+module.exports = {
+	messageDefinitions: {},
+
+	initialize: function(messageDefinitions, useLocale, defaultLocale) {
+		this.defaultLocale = defaultLocale;
+		this.locale = useLocale;
+
+		Object
+			.keys(messageDefinitions)
+			.forEach(function (locale) {
+				this.registerTranslations(locale, messageDefinitions[locale]);
+			}, this);
+	},
+
+	getTranslation: function(token) {
+		var lowerCasedToken = token.toLowerCase();
+
+		return this.messages[lowerCasedToken];
+	},
+
+	getDefaultTranslation: function(token) {
+		var lowerCasedToken = token.toLowerCase();
+
+		return this.defaultMessages[lowerCasedToken];
+	},
+
+	get defaultMessages() {
+		if (this.messageDefinitions[this.defaultLocale] === undefined) {
+			this.messageDefinitions[this.defaultLocale] = {};
+		}
+
+		return this.messageDefinitions[this.defaultLocale];
+	},
+
+	get messages() {
```

Comment:
@briandipalma when running this with closure-medium minifier we get the following error:

ERROR - Parse error. getters are not supported in older versions of JavaScript. If you are targeting newer versions of JavaScript, set the appropriate language_in option.
	get messages() {

This happens in tests and when running the it-app with js.bundle dev-minifier="closure-medium"